css-text/Overview.bs

```diff
@@ -755,6 +755,9 @@ Languages and Typesetting</h4>
          the UA must either <a>hang</a> the <a>white space</a>
          or visually collapse their character advance widths
          such that they don't take up space in the line.
+         However, if 'overflow-wrap' is set to ''break-spaces'',
```

Comment:
By re-reading this, you're modifying an item that I don't understand, so it's not your problem, but you'd have to ask fantasai to review.

This item starts with:
> If spaces or tabs at the end of a line are non-collapsible but have 'white-space' set to ''pre-wrap''

but I can't interpret this part. Does this "but" mean "except"? Maybe clear to native speakers but this is beyond my English capability, sorry.